File: ruby/day08/solution-1.rb
#!/usr/bin/env ruby
# frozen_string_literal: true

input = File.read('input.txt').split("\n")

counter = 0

# Process inputs
input.each do |line|
  output = line.split(' | ')[1].split
  counter += output.count { |o| [2, 3, 4, 7].include?(o.length) }
end

puts counter
# 245

File: typescript/src/day01/solution-1.ts
import * as fs from "fs";

const input = fs
  .readFileSync("../inputs/day01.txt", "utf8")
  .split("\n")
  .map((line) => parseInt(line));

let increases = 0;
for (let i = 0; i < input.length - 1; i++) {
  if (input[i + 1] > input[i]) {
    increases++;
  }
}
console.log(increases);

// Answer: 1482
